temporalio/ext/src/runtime.rs

```diff
@@ -193,6 +206,16 @@ impl Runtime {
             }
         }
     }
+
+    pub fn retrieve_buffered_metrics(&self, durations_as_seconds: bool) -> Result<RArray, Error> {
```

Comment:
Why not store whether durations are seconds on the buffer itself? Feels weird to pass it every time

Reply:
Yeah, no real reason, just seemed to be a conversion-time parameter as opposed to struct state and is how we did it in Python. It is class state from a user POV though.